src/features/events/components/groupedEventsCard.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import moment from 'moment'
+import { FlexColumn } from '../../../components/grid'
+import { media } from '../../../theme/media'
+import { dateFormat, itemsToLoad } from '../../../constants'
+import EventListingCard from './eventListingCard'
+
+const DateGroupHeading = styled.h2`
+  margin: 1rem 0;
+
+  ${media.tablet`
+    display: none;
+  `};
+`
+
+const GroupedEventsCards = ({ event, index, events, toLoad }) => {
+  let header
+  const longDayOfMonth = 'dddd D MMM'
+  let animate
+  // logic to check if the eventCard is not on the original page load or on previuos load
+  if (index >= toLoad - itemsToLoad && toLoad !== itemsToLoad) {
+    animate = true
+  }
```

Comment:
This could be inlined where it is used since it's only assigned then once. Try to avoid the usage of `let` where possible, as it leads to lines of unnecessary code and makes it more difficult to follow the assignment flow.